models/demos/falcon7b/tt/falcon_attention.py

```diff
@@ -317,29 +747,16 @@ def forward(
                     -2,
                     -1,
                     output_mem_config=(
-                        self.model_config["K_TRANSPOSED_OUTPUT_MEMCFG"]
-                        if llm_mode == "prefill" or self.model_config["l1_sharded"] == False
-                        else tt_lib.tensor.MemoryConfig(
+                        tt_lib.tensor.MemoryConfig(
                             tt_lib.tensor.TensorMemoryLayout.HEIGHT_SHARDED, tt_lib.tensor.BufferType.L1
                         )
```

Comment:
this should still be set to `self.model_config["K_TRANSPOSED_OUTPUT_MEMCFG"]` if `self.model_config["l1_sharded"] == False` right?